.changeset/famous-dancers-tan.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': major
+---
+
+unify preview server config
```

Comment:
Can you leave a better explanation? Developers will read the changelog, and I doubt they will understand what "unify" means

Reply:
A more accurate description would be to keep Astro Preview behaving the same way as [Vite Preview](https://vitejs.dev/config/preview-options.html), supporting Vite Preview settings.

According to the Astro code, static previews are supported by default, but third-party adapter previews require additional support.